### PR TITLE
Have Added icons to navbar 

### PR DESCRIPTION
**Pull request by mohak852**
kindly merge this pull request in your final code

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
         return (
             <div className="topnav">
+                <img src={slack} className="slack"/>
                 <a href="https://join.slack.com/t/firstcontributors/shared_invite/enQtNjkxNzQwNzA2MTMwLTVhMWJjNjg2ODRlNWZhNjIzYjgwNDIyZWYwZjhjYTQ4OTBjMWM0MmFhZDUxNzBiYzczMGNiYzcxNjkzZDZlMDM"  target="_blank" rel="noopener noreferrer">Slack</a>
+                <img src={youtube} className="youtube"/>
                 <a href="https://www.youtube.com/channel/UCMXNFxCvyH5LhUwEcmY8qGQ" target="_blank" rel="noopener noreferrer">YouTube</a>
+                <img src={github} className="github"/>
                 <a href="https://twitter.com/1stContribution" target="_blank" rel="noopener noreferrer">Twitter</a>
+                <img src={twitter} className="twitter"/>
                 <a href="https://github.com/firstcontributions/first-contributions" target="_blank" rel="noopener noreferrer">GitHub</a>
```
Comment (sara-02) [inline src/components/Navbar/Navbar.jsx:18]: Looks like you swapped twitter and github.
See the icons https://github.com/firstcontributions/firstcontributions.github.io/issues/10#issuecomment-633080776

Comment (mohak852) [reply src/components/Navbar/Navbar.jsx:18]: I have done those changes but I don't know why they are not reflecting back

Comment (mohak852) [reply src/components/Navbar/Navbar.jsx:18]: So as you a pro in contribution can I ask for help from you

Comment (mohak852) [reply src/components/Navbar/Navbar.jsx:18]: can you please help with my issue

**Comment by mohak852** [comment]
Screenshot from 2020-05-25 05-47-18


It's showing me this error and I have tried almost everything for it. Can you help me to get this

**Comment by mohak852** [comment]
I have changed those icons in Navbar you can go through once.
Screenshot from 2020-05-25 06-00-32

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (Roshanjossey) [reply src/components/Navbar/Navbar.jsx]: You can make changes to the pull request by pushing commit to the same branch it's created from. In this case, ` mohak852:source`

Make sure you've pushed the changes.

**Comment by Roshanjossey** [comment]
@mohak852 , the icons don't look very good. 

This is because the icons are not uniform. Please use an open source/ creative commons source that provides uniform icons. One suggestion https://simpleicons.org/

**Comment by mohak852** [comment]
I have made the necessary changes

**Comment by mohak852** [comment]
actually I have made the changes and made another pull request please can you go through that

**Comment by mohak852** [comment]
Screenshot from 2020-05-26 02-09-42

**Comment by sara-02** [comment]
Screenshot from 2020-05-25 23-51-34
This is what I am talking about.

**Pull request closed without merging**